pybossa/cache/site_stats.py

```diff
@@ -421,3 +421,53 @@ def management_dashboard_stats_cached():
         # reset stats for any missing stat
         map(delete_cache_group, management_dashboard_stats)
     return stats_cached
+
+def total_tasks(interval):
+    """
+    Total tasks
+    """
+    sql = text(f'''
+        SELECT count(task.id) as num_tasks
+        FROM task WHERE
+        to_timestamp(task.created, 'YYYY-MM-DD"T"HH24:MI:SS.US') > current_timestamp - interval '{interval}';
```

Comment:
Curious about the execution plan of this SQL. Could be a good idea to execute it in QA PostgreSQL.